os/net/bluetooth/bluetooth.c

```diff
@@ -71,7 +71,7 @@ struct bt_ad {
 
 /* Work structures: One for high priority and one for low priority work */
 
-static struct work_s g_hip_work;
+static struct work_s g_lowp_work;
```

Comment:
Could you tell me why you change this variable to low?

Reply:
bluetooth subsystem was not getting the response of HCI request from btnull driver as hci_rx_work was pending and init_work was running with hight priority